packages/combobox/stories/combobox.stories.ts

```diff
@@ -145,3 +153,47 @@ export const withHelpText = (): TemplateResult => {
         </sp-combobox>
     `;
 };
+
+class ControlledCombo extends LitElement {
+    static ages: ComboboxOption[] = Array.from({ length: 76 - 55 }, (_, n) => {
+        const age = `${n + 55}`;
+        return { value: age, itemText: age };
+    });
+
+    @state()
+    private value = {
+        raw: '',
+        validated: `${ControlledCombo.ages[0].itemText}`,
+    };
+
+    @query('#age')
+    private combobox!: Combobox;
+
+    override render(): TemplateResult {
+        return html`
+            <sp-field-label for="age">
+                Retirement age (try entering a non-number)
+            </sp-field-label>
+            <sp-combobox
+                id="age"
+                .options=${ControlledCombo.ages}
+                .value=${live(this.value.validated)}
+                @change=${this.onChange}
+            ></sp-combobox>
+        `;
+    }
+
+    private onChange(): void {
+        this.value = {
+            raw: this.combobox.value,
+            validated: this.combobox.value.replace(/\D/g, '') || '55',
+        };
+    }
+}
+defineElement('controlled-combo', ControlledCombo);
+
+export const controlled = (): TemplateResult => {
+    return html`
+        <controlled-combo></controlled-combo>
+    `;
+};
```

Comment:
```suggestion
};
controlled.swc_vrt = {
    skip: true,
};
```
There's not additional VRT data here, so skipping should be fine and will get CI green again.